import-export-cli/integration/apiLogging_test.go

```diff
@@ -0,0 +1,306 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/apim"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
+)
+
+// Get log levels of APIs of the carbon.super tenant in an environment as a super admin user
+func TestGetAPILogLevelsSuperAdminUser(t *testing.T) {
```

Comment:
@binodmx Shall we write these tests in the Table driven format[1], so the code length will be reduced and all user scenarios can be covered (Super tenant Admin, Tenant Admin, Super tenant User and Tenant user)?

[1]. https://github.com/wso2/product-apim-tooling/blob/c8134c2b5948fe4b9edd957696800ef713cb0372/import-export-cli/integration/apiProduct_test.go#L1267-L1322